benchmarks/README.md

```diff
@@ -0,0 +1,64 @@
+# sgkit Benchmarks
```

Comment:
Should we move these instructions to https://pystatgen.github.io/sgkit/latest/contributing.html?